### PR TITLE
Add support for Wandboard Quad (i.MX6) (Lakka-2.3.2)

### PR DESCRIPTION
**Pull request by by-gnome**
Hi-hi,
I made some fixes and added Wandboard support, please check this.

### CODE REVIEW

File: projects/imx6/linux/4.9-wb/linux.arm.conf [hunk 1]
```diff
@@ -0,0 +1,6562 @@
+#
```
Comment (natinusala) [inline projects/imx6/linux/4.9-wb/linux.arm.conf:1]: no aarch64 support?

File: packages/libretro/sameboy/package.mk [hunk 2]
```diff
@@ -19,7 +19,7 @@
 ################################################################################
 
 PKG_NAME="sameboy"
-PKG_VERSION="54f41cb"
+PKG_VERSION="44d8b16"
```
Comment (gouchi) [inline packages/libretro/sameboy/package.mk:22]: We should address this issue with this [PR](https://github.com/libretro/Lakka-LibreELEC/pull/958) ? As it is not directly linked to adding support for Wandboard Quad (i.MX6).

**Comment by gouchi** [comment]
Thank you for adding support for this board.

Looking forward to seeing  [etna_viv](https://github.com/etnaviv/etna_viv) in the future ;-)

File: packages/graphics/imx-gpu-g2d/package.mk [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with OpenELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="imx-gpu-g2d"
+if [ "$LINUX" = "imx6-4.9-wb" ]; then
+  PKG_VERSION="6.2.2.p0"
```
[the remark this answers is not in the document]
Comment (by-gnome) [reply packages/graphics/imx-gpu-g2d/package.mk:21]: I agree

File: packages/tools/u-boot/release [hunk 4]
```diff
@@ -33,7 +33,11 @@ mkdir -p $RELEASE_DIR/3rdparty/bootloader
   cp -PR $BUILD/u-boot-*/u-boot*.imx $RELEASE_DIR/3rdparty/bootloader 2>/dev/null || :
   cp -PR $BUILD/u-boot-*/u-boot*.img $RELEASE_DIR/3rdparty/bootloader 2>/dev/null || :
 
-  cp -PR $PROJECT_DIR/$PROJECT/bootloader/uEnv*.txt $RELEASE_DIR/3rdparty/bootloader 2>/dev/null || :
+  if [ "$SYSTEM" = "wandboard" ]; then
```
[the remark this answers is not in the document]
Comment (by-gnome) [reply packages/tools/u-boot/release:36]: I hurried, this change is not required.
Will delete it.

File: projects/imx6/linux/4.9-wb/linux.arm.conf (hunk 1 above)
Comment (by-gnome) [reply projects/imx6/linux/4.9-wb/linux.arm.conf:1]: I don't shure about imx6, this linux and Aarch64.

File: packages/libretro/sameboy/package.mk (hunk 2 above)
Comment (by-gnome) [reply packages/libretro/sameboy/package.mk:22]: This is just fix for Lakka-2.3.2.

**Comment by by-gnome** [comment]
> Thanks for the PR! Did you try other IMX targets to make sure they still work?

Other imx6 targets (cuboxi, udoo, matrix) are not built from repo because [this](https://patchwork.kernel.org/patch/10546809/) and u-boot needs compiler-gcc9.h
I can add these patches to the next commit, or somehow.
How can i change this PR?

**Comment by by-gnome** [comment]
> Look forward to seeing [etna_viv](https://github.com/etnaviv/etna_viv) in the future ;-)

Thx.

File: packages/libretro/sameboy/package.mk (hunk 2 above)
Comment (by-gnome) [reply packages/libretro/sameboy/package.mk:22]: I don't know, you say.

Comment (by-gnome) [reply packages/libretro/sameboy/package.mk:22]: Strange things happens in buildbot branch of libretro/SameBoy.
Few time ago i used libretro/SameBoy@c9e547c, but now it out of branch like libretro/SameBoy@54f41cb.
Look [this ](https://github.com/libretro/Lakka-LibreELEC/pull/926) please.

Comment (natinusala) [reply packages/libretro/sameboy/package.mk:22]: It's normal, the repo is regularly rebased against upstream emulator and it breaks commits ids every time a rebase is done

**Comment by by-gnome** [comment]
I will add patches for _linux-4.4-xbian_ and _cuboxi u-boot_ to fix _cuboxi, udoo and matrix_ compilation with gcc-9 in next PR.

**Comment by by-gnome** [comment]
BT and WiFi don't work on Wandboard, will try to fix.

**Comment by Ntemis** [comment]
you still need this?

**Comment by Ntemis** [comment]
i saw you migrated to master so am closing this one down too, no need to beat a dead horse imo

**Pull request closed without merging**